### PR TITLE
Fix ignored "log_output" option

### PR DESCRIPTION
**Pull request by joeldavidw**
Added a fix for the issue where ```log_output``` was being ignored and not populated into configurations.yaml.

#### Checklist
- [x] PR done against `dev` branch
- [x] Updated `common/Dockerfile` with new version
- [x] Updated `zigbee2mqtt/CHANGELOG.md` file with new changes
- [x] Updated `zigbee2mqtt/DOCS.md` with any changes
- [x] Add any new configuration options to `zigbee2mqtt/config.json` and `zigbee2mqtt-edge/config.json`.

### CODE REVIEW

**Comment by joeldavidw** [comment]
## Issue Details

**Description of problem:**
**Zigbee2mqtt add-on version:**
```
1.18.1-1
```

**Your entire configuration from the frontend**:
```
advanced:
  log_output:
    - console
```

**Running platform**
```
Core: core-2021.3.2
Supervisor: supervisor-2021.03.4
Operating System: HassOS 5.12
```

**Your logs from Home Assistant Supervisor**
Zigbee2mqtt Log
```
Zigbee2MQTT:info  2021-03-06 15:56:09: Logging to console and directory: '/config/zigbee2mqtt/log/2021-03-06.15-56-08' filename: log.txt
```
Supervisor Log
```
21-03-06 07:55:59 WARNING (SyncWorker_0) [supervisor.addons.options] Unknown option 'log_output' for Zigbee2mqtt
```

**Comment by ciotlosm** [comment]
Thanks. I will have a look tomorrow.

File: zigbee2mqtt/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
+## 1.18.1-2
+- ⚠️ **Breaking changes**
+  - `log_output` now required option in `advanced`
+- Fix `log_output` option being ignored
+
+Default zigbee2mqtt options for `log_output`
+```
+advanced:
```
Comment (ciotlosm) [inline zigbee2mqtt/CHANGELOG.md:8]: I don't think an example should be added into the changelog. Better to either link to official docs or don't mention it if it's seamless for most users.

File: zigbee2mqtt/config.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Zigbee2mqtt",
-  "version": "1.18.1-1",
+  "version": "1.18.1-2",
```
Comment (ciotlosm) [inline zigbee2mqtt/config.json:3]: Can you please update to 1.18.1-3? I've just merged another PR for ezsp adapter.

**Review by ciotlosm — COMMENTED** [review]
Did you run a test locally as well?

File: zigbee2mqtt/CHANGELOG.md (hunk 1 above)
Comment (joeldavidw) [reply zigbee2mqtt/CHANGELOG.md]: I have removed the example and linked to official docs instead. Correct me if I am wrong but users who did not state log_output option previously would not be able to start the add-on or save a new config without the log_output option.

**Comment by joeldavidw** [comment]
Yep, I have tested and the added fix will allow users to set their desired log output mode.

#### Configuration
```
log_output:
  - console
```

#### Output After Fix
```
Zigbee2MQTT:info  2021-03-09 01:04:55: Logging to console only'
```

File: zigbee2mqtt/CHANGELOG.md (hunk 1 above)
Comment (ciotlosm) [reply zigbee2mqtt/CHANGELOG.md]: If it's not an optional config, you might be correct. Best to test this before releasing.

Comment (ciotlosm) [reply zigbee2mqtt/CHANGELOG.md]: Can you test also how a migration would work? It's best to check if we break the setup for existing users without this option.

Comment (joeldavidw) [reply zigbee2mqtt/CHANGELOG.md]: > If it's not an optional config, you might be correct. Best to test this before releasing.

This is not an optional config as it requires at least an empty array.


> Can you test also how a migration would work? It's best to check if we break the setup for existing users without this option.

Yep, I have tested this and without this option the add-on would not be able to start and existing users would be prompted the message below.

Screenshot 2021-03-09 at 2 31 51 AM

**Pull request closed without merging**